Awesome/styles2.css

```diff
@@ -0,0 +1,536 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&family=Raleway:wght@100;200;300;400;700&display=swap');
+.awesome{
+  font-family: "Raleway";
+  font-weight: normal;
+}
+
+.header{
+  background: #000000;
+  display: flex;
+  justify-content: space-between;
+}
+.header img{
```

Comment:
It's better to add class for logo image.

Reply:
corrected